docs/console-api/daily-reports-depricated.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Daily Reports Depricated
+sidebar_label: Daily Reports Depricated
+slug: /console-api/daily-reports-depricated
+description: Download daily reports depricated
+# hide_table_of_contents: true
+custom_edit_url: null
+---
+
+# Deprecated: v1/get_daily_report
+
+## Authorization
+
+All requests must include the **Server-Side Secret Key** field in the header. This can can be created in the Project Settings on [console.statsig.com/api_keys](https://console.statsig.com/api_keys). This feature requires Pro or Enterprise Tier.
+
+:::info
+This is available for Enterprise contracts and needs to be enabled for you, once. Please reach out to our [support team](mailto:support@statsig.com), your sales contact, or via our slack channel if you want this enabled.
+:::
```

Comment:
Since its deprecated, we wont be adding anyone to this endpoint any more.  So I'd update this messaging